### DISCUSSION TITLE
Screen randomly goes black

### DISCUSSION THREAD

**Discussion by buraienn** [rocksdanister/lively#1900] [General]
Screen randomly goes black when I am trying to navigate between folders or switching between different programs like chrome etc.
[lively_log_20230919_102003.zip](https://github.com/rocksdanister/lively/files/12655065/lively_log_20230919_102003.zip)

**Comment by buraienn** [comment]
Currently using the latest version downloaded from microsoft store

**Comment by MikeNeoMike** [comment]
Bei mir beendet sich auch das Video selbstständig und lässt sich auch nicht wieder starten. Erst nach einen Neustart von Windows geht es wieder. Das Problem trat nach den neuen Funktionsupdate von Windows 11 auf.
Hat eventuell jemand eine Lösung dazu?

[lively_log_20231011_181352.zip](https://github.com/rocksdanister/lively/files/12872256/lively_log_20231011_181352.zip)

**Comment by rocksdanister** [comment]
dup: https://github.com/rocksdanister/lively/issues/1926
